### PR TITLE
Don't show env when there isn't

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
This PR removes unneeded and odd Skeletons.

image
image
image

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/web/src/components/application-detail-page/application-detail/index.tsx [hunk 1]
```diff
@@ -213,7 +213,7 @@ export const ApplicationDetail: FC<ApplicationDetailProps> = memo(
               {app ? app.name : <Skeleton width={100} />}
             </Typography>
             <Typography variant="subtitle2" className={classes.env}>
-              {env ? env.name : <Skeleton width={100} />}
+              {env && env.name}
             </Typography>
```
Comment (khanhtc1202) [inline pkg/app/web/src/components/application-detail-page/application-detail/index.tsx:217]: nits,
```typescript
{env && (
  <Typography variant="subtitle2" className={classes.env}>
    {env.name}
  </Typography>
)}
```

Comment (nakabonne) [reply pkg/app/web/src/components/application-detail-page/application-detail/index.tsx:217]: Right. The same as the deployment detail page.

Comment (nakabonne) [reply pkg/app/web/src/components/application-detail-page/application-detail/index.tsx]: Fixed

**Comment by khanhtc1202** [comment]
Nice, just the `web-test` on CI's taking so long
/approve

**Pull request merged**